File: 01_circle_area/code.java
// Online Java Compiler
// Use this editor to write, compile and run your Java code online
import java.util.Scanner;
class Main {

    static double calculateCircleArea(double radius) {
        double area = radius * radius * Math.PI;
        return area;
    }

    public static void main(String[] args) {

        Scanner scanner  = new Scanner(System.in);

        System.out.println("Enter The Radius");

        double radius = scanner.nextDouble();

        double area = calculateCircleArea(radius);

        System.out.println("Circle Area" + area);

        scanner.close();

    }
}

File: 01_circle_area/code.cpp
#include <iostream>
using namespace std;

double calculateCircleArea(double radius) 
{
    double area = 3.14 * radius * radius;

    return area;
}

int main()
{
    cout << "Enter the radius:";
    double radius;
    cin >> radius;

    double area = calculateCircleArea(radius);
    cout << "Circle Area =" << area << endl;
    return 0;
}
